spec/factories/checkins.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :checkin do
+    
+  end
+end
```

Comment:
Well these were added in when I created the model I wasn't sure if they should be left in for future use or deleted, but I'll go ahead and go through and delete all of the extra unused files